modules/events_archive/EED_Events_Archive.module.php

```diff
@@ -227,21 +218,19 @@ public function run($WP)
         EEH_Event_Query::add_query_filters();
         // set params that will get used by the filters
         EEH_Event_Query::set_query_params(
-            '',    // month
-            '',    // category
-            $config->display_expired_events,    // show_expired
-            'start_date',    // orderby
-            'ASC'    // sort
+            '',                             // month
+            '',                             // category
+            $config->display_expired_events // show_expired
```

Comment:
those last two params were removed because they were the same as the default values for that function